docs/integrations/databases/opentelemetry/redis-opentelemetry.md

```diff
@@ -0,0 +1,144 @@
+
+Once the logs are configured to write to a local file, follow the steps below to configure the collection in Sumo Logic.
+
+## Configuring Collection and Installing the Redis App
+
+As part of setting up the collection process and app installation, you can select the App from **App Catalog** and click **Install App**. Follow the steps below:
+
+### Step 1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the "Use an existing Collector" option.
+:::
+
+To create a new Collector:
+
+1. Select the **Add a new Collector** option.
+1. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent.<br/><img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to the Sumo Logic platform.
+
+In this step, you will configure the yaml file required for Redis Collection.
+
+The log file path configured to capture redis logs must be given here. The files are typically located in `/var/log/redis/redis-server.log`. If you are using a customized path, check the [`redis.conf`](https://download.redis.io/redis-stable/redis.conf) file for this information.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/configuration-screenshot.png')} alt="Configuration" />
+
+
+### Step 3: Sending logs to Sumo Logic
+
+Once you've downloaded the yaml file (in step 2), follow the below steps based on your environment.
+
+<Tabs
+  className="unique-tabs"
+  defaultValue="Linux"
+  values={[
+    {label: 'Linux', value: 'Linux'},
+    {label: 'macOS', value: 'macOS'},
+  ]}>
+
+<TabItem value="Linux">
+
+1. Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance that needs to be monitored.
+2. Restart the otelcol-sumo process using:
+  ```sh
+  sudo systemctl restart otelcol-sumo
+  ```
+
+</TabItem>
+<TabItem value="macOS">
+
+1. Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance that needs to be monitored.
+2. Restart the otelcol-sumo process using:
+  ```sh
+  otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+  ```
+
+</TabItem>
+</Tabs>
+
+Sumo Logic will start receiving data from your host machine when the above command is successfully executed. This will install the app and dashboards to your Sumo Logic org.
```

Comment:
```suggestion
Sumo Logic will start receiving data from your host machine when the above command is successfully executed. 
Press **Next** . This will install the app and dashboards to your Sumo Logic org.
```